bionic/code_hasher.py

```diff
@@ -219,6 +219,19 @@ def _update_hash(self, hash_accumulator, obj, code_context):
                 obj_bytes=self._check_and_hash(obj.value, code_context),
             )
 
+        elif isinstance(obj, (property, types.DynamicClassAttribute)):
+            if isinstance(obj, list):
+                type_prefix = TypePrefix.PROPERTY
+            else:
+                type_prefix = TypePrefix.DYNAMIC_CLASS_ATTR
+            # A property is identified using fget, fset, fdel, and doc.
+            members = [obj.fget, obj.fset, obj.fdel, obj.__doc__]
+            add_to_hash(
+                hash_accumulator,
+                type_prefix=TypePrefix.HASH,
```

Comment:
Should this be `type_prefix=type_prefix`?

Reply:
Yes 🤦🏽

Reply:
ah with the current pattern, this should be `TypePrefix.Hash`. But I'm missing `add_to_hash(hash_accumulator, type_prefix=type_prefix)` before this.